hyperspy/drawing/mpl_he.py

```diff
@@ -179,6 +180,29 @@ def plot(self, **kwargs):
             self.plot_navigator(**kwargs.pop('navigator_kwds', {}))
         self.plot_signal(**kwargs)
 
+        if "ipympl" in mpl.get_backend() and preferences.Plot.enable_ipympl_plotting:
+            # Then we can use the widget backend for two figures horizontally
+            plot_style = preferences.Plot.ipympl_plot_style
+            if plot_style != "hidden":
```

Comment:
I assume you're referring to `plot_style == "hidden"`? 
I added it as an afterthought - it means that anyone making some sort of "custom plotting tool" where they maybe link several figures, could for instance link several signals through the interactive interfaces (ROI / Events), and then have the figures appear in a nice layout of their creation.

Of course, since it's a global setting, plotting some other signal would also be hidden, so this definitely isn't the best way to do this. I'm just experimenting. Perhaps something like `s.plot(navigator_output_widget=output_widget1, signal_output_widget = output_widget2)` is a better alternative?

Reply:
I've got a much better implementation coming now.